.github/workflows/release-python-osx.yml

```diff
@@ -0,0 +1,51 @@
+name: Build pymongoarrow wheels on OSX
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build-osx-wheels:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-10.15]
```

Comment:
`macos-11.0` is not available for public consumption at this time even though GH docs say that it is. See [this post](https://github.com/actions/virtual-environments/issues/2486) for details.